docs/pages/guides/hello-world/filter-sync.mdx

```diff
@@ -119,14 +118,14 @@ export async function setupNetwork() {
     startBlock: BigInt(networkConfig.initialBlockNumber),
     filters: [
       {
-        tableId: resourceToHex({ type: "table", namespace: "", name: "Counter" }),
+        tableId: config.tables.app__Counter.tableId
       },
       {
-        tableId: resourceToHex({ type: "table", namespace: "", name: "History" }),
+        tableId: resourceToHex({ type: "table", namespace: "app", name: "History" }),
         key0: pad("0x01"),
       },
       {
-        tableId: resourceToHex({ type: "table", namespace: "", name: "History" }),
+        tableId: resourceToHex({ type: "table", namespace: "app", name: "History" }),
```

Comment:
there are many other table IDs in this file that still need updating to the new pattern

Reply:
Oops, sorry. I was too tired. Fixed now, but it's enough of a change I should rerun the tutorial to be sure.